package.json

```diff
@@ -52,16 +52,16 @@
     "react-native-keychain": "4.0.5",
     "react-native-linear-gradient": "2.5.6",
     "react-native-local-auth": "1.6.0",
-    "react-native-navigation": "6.3.0",
-    "react-native-notifications": "2.0.6",
+    "react-native-navigation": "6.3.3",
+    "react-native-notifications": "2.1.7",
     "react-native-passcode-status": "1.1.2",
     "react-native-permissions": "2.0.10",
     "react-native-safe-area": "0.5.1",
     "react-native-section-list-get-item-layout": "2.2.3",
     "react-native-slider": "0.11.0",
     "react-native-status-bar-size": "0.3.3",
     "react-native-svg": "12.0.3",
-    "react-native-v8": "0.61.5-patch.3",
+    "react-native-v8": "github:mattermost/react-native-v8#v0.62.0",
```

Comment:
I've submitted a PR https://github.com/Kudo/react-native-v8/pull/41 to update v8 to RN 62